src/utils.js

```diff
@@ -288,6 +288,10 @@ export function findSelectorInTree(selectors, tree, selectFirst = false, searchF
                 if (typeof child.name === 'string') {
                     return matchSelector(selector, child.name)
                 } else if (child.name !== null && typeof child.name === 'object') {
+                    if (child.name.render !== undefined && child.name.render.name !== null &&
+                        typeof child.name.render.name === 'string') {
```

Comment:
```suggestion
                    if (child.name.render && child.name.render.name) {
```

I don't think there's a need to check if the name is a string.